src/Neo/ProtocolSettings.cs

```diff
@@ -69,7 +69,7 @@ public record ProtocolSettings
         /// <summary>
         /// The maximum increment of the <see cref="Transaction.ValidUntilBlock"/> field.
         /// </summary>
-        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock;
+        public uint MaxValidUntilBlockIncrement => 86400000 / MillisecondsPerBlock; //TODO keep the same??
```

Comment:
This one is configurable per-network, so I'd rather leave the default as is and configure particular networks.

Reply:
ok